### ISSUE TITLE
Species as synonyms of family

### ISSUE THREAD

**Issue by snacktavish** [OpenTreeOfLife/reference-taxonomy#348]
https://tree.opentreeoflife.org/taxonomy/browse?id=5603
These synonyms coming in from gbif are synonymizing species names with a family, which I'm pretty sure should never happen.

**Comment by TonyRees** [comment]
Looks like there is something odd at the Paleobiodb end, see
http://fossilworks.org/bridge.pl?a=taxonInfo&taxon_no=37674 , with a lot of
things listed as "sister taxa" to the family that should be subordinates
(and of course species names should be subordinate to their containing
genera - or sometimes subgenera - anyway). I do not know how well
controlled the PBDB taxonomic hierarchy is, or whether this is an example
of one or many such cases...

- Tony

Tony Rees, New South Wales, Australia
https://about.me/TonyRees

**Comment by jar398** [comment]
I agree that having 'sister taxa' of lower rank is annoying; usually they are collected together into an Incertae sedis container. But I see nothing wrong with Paleobiodb itself, assuming that interpretation; they just have a pile of turtle fossils and don't know what families to put them in.

The GBIF version of Testudines in Paleobiodb is here: https://www.gbif.org/species/121459769 and it shows the sister taxa as being converted to synonyms, which I think is the real bug. The error is carried over to the GBIF backbone, and from there to OTT.

My experience is that algorithmic errors like this are usually replicated in many different places. For example, you see the same thing in Reptilia in GBIF's paleobiodb import: https://www.gbif.org/species/121459544  although oddly *not* in GBIF's Reptilia https://www.gbif.org/species/358 .

**Comment by snacktavish** [comment]
I agree that it is usually a deeper problem but this doesn't actually seem *too* common.
This turned up as part of  mapping VTO identifiers to OTT_ids at https://github.com/phenoscape/KB-DataFest-2017/wiki
@pasanfernando ran the counts of how many  OTT ids had more than one VTO id:
number of mappings | count
2 | 1652
3 | 126
4 | 26
5 | 10
6 | 4
7 | 2
8 | 1
10 | 1 (I think this is the taxon linked in this issue )

We ran out of time to dig into them in detail, (many are surely correct synonyms) but will clean things up and post better documentation at: https://github.com/phenoscape/PhyloPheno

We will submit this issue, and other ones that turn up to GBIF

**Comment by snacktavish** [comment]
https://github.com/gbif/checklistbank/issues/50

**Comment by snacktavish** [comment]
https://github.com/gbif/checklistbank/issues/50
